azure-servicebus/pom.xml

```diff
@@ -45,6 +45,13 @@
 		</plugins>
 	</build>
 
+	<repositories>
+		<repository>
+			<id>azure eventhubs</id>
```

Comment:
Whats this for?

Reply:
That's used to access the preview version of qpid-proton-j-extensions, but that might be able to be replaced with an official version now. I'll check that and get back to you.